src/main/app/src/interfaces.ts

```diff
@@ -1,10 +1,35 @@
 export interface ILandscape {
-    name: string;
-    identifier: string;
-    description?: string;
-    contact?: string;
-    teams: string[];
-    groups?: string[];
-    items?: string[];
-    lastUpdate?: string;
+  name: string;
+  identifier: string;
+  description?: string;
+  teams?: object[]; // TODO: Create interface
+  items?: IItem[];
+  groups?: object[]; // TODO: Create interface
+  lastUpdate?: string;
+  overallState?: string;
```

Comment:
see LandscapeDTO, I think overallState has been removed

Reply:
yeah i wasn't sure if it was in before because we always had no data for it, i'll remove it for now